popup.js

```diff
@@ -12,12 +12,31 @@ function sendRequest(data) {
 }
 
 jQuery(document).ready(function($) {
-    //send message request to content script
-    $('#btnUpload').click(function() {
-        sendRequest({
-            type: 'addJson',
-            jsontext: $('#jsontext').val()
-        });
+    $('#form-import').submit(function(e) {
```

Comment:
@J2TeaM  A longtime I wasn't use `submit` (i was a bad things :) ), and anything else reason for this approach?.
Can we comunicate this project via Facebook?, this is my facebook : https://www.facebook.com/jodinh.duong .